src/pages/Pools/Pools.tsx

```diff
@@ -72,6 +75,8 @@ function Pools(): ReactElement | null {
     })
   }, [account, chainId])
 
+  if (isConnecting) return <div> Loading ...</div>
```

Comment:
If you're going to display a loading state, can you create a better component for it than this?

Reply:
I added this while debugging of `underlying netowork connection`, but these are not related that issue and we don't have much need to displaying loading component while connect account. So, I'll remove this lines.